stdlib/public/core/Codable.swift

```diff
@@ -67,6 +67,12 @@ public protocol CodingKey {
     init?(intValue: Int)
 }
 
+extension CodingKey {
+    public var description: String {
```

Comment:
Please add a doc comment describing this (something like "A textual representation of this key." should suffice).